### DISCUSSION TITLE
Hiding SummaryRow

### DISCUSSION THREAD

**Discussion by taltshaked** [ant-design-blazor/ant-design-blazor#2556] [Q&A]
Hi,
Is there a way to toggle the summary row on and off (bind it to a switch)?
I tried the Hidden property in SummaryCell and it does nothing.
Also tried an "if" block around the entire SummaryRow and SummaryCell but each time I toggle the show/hide state, a new summary row is added instead of hiding or showing a single summary row.

**Comment by ElderJames** [comment]
Hey，Maybe this is a bug. We can create an issue.
